src/components/app/App.js

```diff
@@ -23,6 +24,7 @@ function App() {
                 <Route path={'/'} exact component={Content} />
                 <Route path={'/login'} exact component={LoginPage} />
                 <Route path="/card/:id" component={SingleCard} />
+                <Route path="/settings" component={Settings} />
```

Comment:
Settings route should be accessible only for authorized admin user. Right now it's accessible on any case, just link to it is not rendered.